### PR TITLE
Bug/claudia 5158

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg @jframos 
https://jirapdi.tid.es/browse/CLAUDIA-5158
Testbed locally against FILAB in a concrete error

### CODE REVIEW

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java [hunk 1]
```diff
@@ -123,9 +123,9 @@ public TierInstance create(ClaudiaData data, String envName, TierInstance tierIn
             tierInstance.setTier(tierDB);
             createSecurityGroups(data, tierInstance);
         } catch (InvalidSecurityGroupRequestException isgre) {
-            String secGroupMen = "The securityGroupRequest creation is invalid"; 
+            String secGroupMen = "The securityGroupRequest creation is invalid " + isgre.getMessage();
             log.error(secGroupMen);
-            throw new InvalidEntityException(secGroupMen + " : " + isgre.getMessage());
+            throw new InvalidEntityException(secGroupMen, isgre);
         } catch (EntityNotFoundException enfe) {
             String men = "ProductRelease in tier " + tierInstance.getTier().getName() + "does not exist";
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java]: add a blank space after tierInstance.getTier().getName() or change the final sentence by " does not exist"

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java [hunk 2]
```diff
@@ -123,9 +123,9 @@ public TierInstance create(ClaudiaData data, String envName, TierInstance tierIn
             tierInstance.setTier(tierDB);
             createSecurityGroups(data, tierInstance);
         } catch (InvalidSecurityGroupRequestException isgre) {
-            String secGroupMen = "The securityGroupRequest creation is invalid"; 
+            String secGroupMen = "The securityGroupRequest creation is invalid " + isgre.getMessage();
             log.error(secGroupMen);
```
Comment (jesuspg) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java]: change log.error trace by log.warning

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java [hunk 3]
```diff
@@ -123,9 +123,9 @@ public TierInstance create(ClaudiaData data, String envName, TierInstance tierIn
...
             createSecurityGroups(data, tierInstance);
         } catch (InvalidSecurityGroupRequestException isgre) {
-            String secGroupMen = "The securityGroupRequest creation is invalid"; 
+            String secGroupMen = "The securityGroupRequest creation is invalid " + isgre.getMessage();
             log.error(secGroupMen);
-            throw new InvalidEntityException(secGroupMen + " : " + isgre.getMessage());
+            throw new InvalidEntityException(secGroupMen, isgre);
         } catch (EntityNotFoundException enfe) {
             String men = "ProductRelease in tier " + tierInstance.getTier().getName() + "does not exist"; 
             log.error(men);
```
Comment (jesuspg) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/TierInstanceManagerImpl.java]: change log.error by log.warning

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**